<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingParamDialogClass</class>
 <widget class="QDialog" name="SettingParamDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SettingParamDialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidgetAllParam">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>421</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabGeneral">
    <attribute name="title">
     <string>General Param</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabComm">
    <attribute name="title">
     <string>Comm</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabCamStrategy">
    <attribute name="title">
     <string>Cam Strategy</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabResult">
    <attribute name="title">
     <string>Result</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidgetResult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>611</width>
       <height>371</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Result</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabAlgorithm">
    <attribute name="title">
     <string>Algorithm</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidgetAlgorithm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>611</width>
       <height>381</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>ページ</string>
      </attribute>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabOutlineOffset">
    <attribute name="title">
     <string>OutlineOffset</string>
    </attribute>
    <widget class="QPushButton" name="ButtonOutlineOffset">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>340</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
     <property name="icon">
      <iconset resource="ButtonSettingParameters.qrc">
       <normaloff>:/Qtres/Save.png</normaloff>:/Qtres/Save.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabGUI">
    <attribute name="title">
     <string>GUI</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="Direction">
    <attribute name="title">
     <string>Direction</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Scan phase</string>
    </attribute>
    <widget class="QFrame" name="frameScanPhase">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>491</width>
       <height>371</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Image table</string>
    </attribute>
    <widget class="QPushButton" name="pushButtonRowImageTable">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>201</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Row image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonLoadImageTable">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>201</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Load image table</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="frameBottom">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>420</y>
     <width>641</width>
     <height>61</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="ButtonCancel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
    <property name="icon">
     <iconset resource="ButtonSettingParameters.qrc">
      <normaloff>:/Qtres/Cancel.png</normaloff>:/Qtres/Cancel.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonOK">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>OK  Save</string>
    </property>
    <property name="icon">
     <iconset resource="ButtonSettingParameters.qrc">
      <normaloff>:/Qtres/Save.png</normaloff>:/Qtres/Save.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEachPage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Each page</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ButtonSettingParameters.qrc"/>
 </resources>
 <connections/>
</ui>
